ui-participant/src/hub/HubPage.tsx

```diff
@@ -69,7 +100,36 @@ export default function HubPage() {
           className="hub-dashboard py-4 px-2 px-md-5 my-md-4 mx-auto shadow-sm"
           style={{ background: '#fff', maxWidth: 768 }}
         >
-          {enrollees.map(enrollee => <StudySection key={enrollee.id} enrollee={enrollee} portal={portal} />)}
+          <div className="dropdown hub-dashboard-background flex-grow-1">
+            <button
+              className="btn btn-outline-primary dropdown-toggle w-100 position-relative d-flex justify-content-center
+              align-items-center" type="button"
+              data-bs-toggle="dropdown" aria-expanded="false" id="dropdownMenuButton">
+              Participants and Tasks
+              <span className={` alert-circle position-absolute rounded-circle
+                    ${getTotalTasks() == 0 ? 'bg-secondary-subtle' : 'bg-danger text-white'}`}
+              style={{ right: '12px', top: '50%', transform: 'translateY(-50%)' }}>
+                {getTotalTasks()}
+              </span>
+            </button>
+
+
+            <ul className="dropdown-menu w-100" aria-labelledby="dropdownMenuButton">
+
+              <HubPageParticipantSelector enrollee={enrollees[0]} profile={profile}
+                relationshipType={undefined}/>
+
+              {relations?.map(enrolleeRelation => (
+                <HubPageParticipantSelector enrollee={enrolleeRelation.relation.targetEnrollee}
+                  profile={enrolleeRelation.profile}
+                  relationshipType={enrolleeRelation.relation.relationshipType}/>
+              ))}
```

Comment:
These selector options need a unique key (otherwise they'll display an error in the browser console logs). Something like this should do it:

```suggestion
              {relations?.map((enrolleeRelation, index) => (
                <HubPageParticipantSelector key={index} enrollee={enrolleeRelation.relation.targetEnrollee}
                  profile={enrolleeRelation.profile}
                  relationshipType={enrolleeRelation.relation.relationshipType}/>
              ))}
```